TypeScript.YAML-tmLanguage

```diff
@@ -273,8 +273,10 @@ repository:
     end: '\)'
     patterns:
     - include: '#comment'
+    - include: '#parameter-name'
```

Comment:
This looks like a non-trivial change that could have quite an impact.  Let's chat about this in person tomorrow before you merge anything.